src/main/resources/default/taglib/t/navboxLink.html.pasta

```diff
@@ -4,16 +4,20 @@
 <i:arg name="active" type="boolean" default="false"/>
 <i:arg name="permission" type="String" default=""/>
 <i:arg name="icon" type="String" default=""/>
+<i:arg name="class" type="String" default=""/>
```

Comment:
Maybe name it "labelClass" so other class-attributes could later be added to alter the link or icon, ...